cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java

```diff
@@ -20,14 +23,24 @@
  * Cruise Control will use {@link HttpSession} to keep track the progress of such requests and
  * report the progress to the users.
  */
+@JsonResponseClass
 public class OperationProgress {
-  private static final String STEP = "step";
-  private static final String DESCRIPTION = "description";
-  private static final String TIME_IN_MS = "time-in-ms";
-  private static final String COMPLETION_PERCENTAGE = "completionPercentage";
+  @JsonResponseField
+  protected static final String OPERATION = "operation";
+  @JsonResponseField
+  protected static final String OPERATION_PROGRESS = "operationProgress";
   private boolean _mutable = true;
   private List<OperationStep> _steps = new ArrayList<>();
   private List<Long> _startTimes = new ArrayList<>();
+  private String _operation;
+
+  public  OperationProgress() {
+    this("");
```

Comment:
This is to keep the old API and it is used in when we start an async job internally(e.g. from [self-healing ](https://github.com/linkedin/cruise-control/blob/3b0ea9cc47fc7c73c009ca9bb7903d5a1ffbfa3d/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/GoalViolationDetector.java#L153)). Here we do not care the operation name since it is invisible from client.